hnn_core/cell.py

```diff
@@ -105,73 +105,46 @@ def create_soma(self):
         self.soma.Ra = soma_props['Ra']
         self.soma.cm = soma_props['cm']
 
-    def get3dinfo(self):
-        """Get 3d info."""
-        ls = self.get_sections()
-        lx, ly, lz, ldiam = [], [], [], []
-        for s in ls:
-            for i in range(s.n3d()):
-                lx.append(s.x3d(i))
-                ly.append(s.y3d(i))
-                lz.append(s.z3d(i))
-                ldiam.append(s.diam3d(i))
-        return lx, ly, lz, ldiam
-
-    def getbbox(self):
-        """Get cell's bounding box."""
-        lx, ly, lz, ldiam = self.get3dinfo()
-        minx, miny, minz = 1e9, 1e9, 1e9
-        maxx, maxy, maxz = -1e9, -1e9, -1e9
-        for x, y, z in zip(lx, ly, lz):
-            minx = min(x, minx)
-            miny = min(y, miny)
-            minz = min(z, minz)
-            maxx = max(x, maxx)
-            maxy = max(y, maxy)
-            maxz = max(z, maxz)
-        return ((minx, maxx), (miny, maxy), (minz, maxz))
-
-    def translate3d(self, dx, dy, dz):
-        """Translate 3d."""
-        for s in self.get_sections():
-            for i in range(s.n3d()):
-                h.pt3dchange(i, s.x3d(i) + dx, s.y3d(i) + dy,
-                             s.z3d(i) + dz, s.diam3d(i), sec=s)
-
-    def translate_to(self, x, y, z):
-        """Translate to position."""
+    def move_to_pos(self):
+        """Move cell to position."""
         x0 = self.soma.x3d(0)
         y0 = self.soma.y3d(0)
         z0 = self.soma.z3d(0)
-        dx = x - x0
-        dy = y - y0
-        dz = z - z0
-        # print('dx:',dx,'dy:',dy,'dz:',dz)
-        self.translate3d(dx, dy, dz)
+        dx = self.pos[0] * 100 - x0
+        dy = self.pos[2] - y0
+        dz = self.pos[1] * 100 - z0
 
-    def move_to_pos(self):
-        """Move cell to position."""
-        self.translate_to(self.pos[0] * 100, self.pos[2], self.pos[1] * 100)
```

Comment:
I don't quite get this ... why `pos[2]` for `y` and why not `pos[2] * 100`

Reply:
This is confusing. Why `pos[2]` instead of `pos[1]` for the y-coordinate?

Is the factor of `100` supposed to represent the actual distance between cells (i.e., scales the integer positions within the coordinate grid)?

Reply:
I honestly don't know ... I raised an issue https://github.com/jonescompneurolab/hnn-core/issues/202. Keeping it unchanged for now. Just reorganized the code.